dnf/util.py

```diff
@@ -25,22 +25,186 @@
 from .pycomp import PY3, basestring
 from functools import reduce
 
+
+##from dnfpluginscore import _, logger
+from dnf.i18n import _
+import dnf
+import iniparse
+
 import dnf.const
 import dnf.pycomp
 import itertools
 import librepo
+import logging
 import os
 import pwd
 import shutil
 import subprocess
 import tempfile
 import time
 
-"""DNF Utilities.
+logger = logging.getLogger('dnf')
+
+"""DNF Utilities."""
+
+
+def get_reposdir(plugin):
```

Comment:
This could be moved into `dnf.conf.Conf` too. It could be a method with no params then.